projects/network_tester/src/supervisor.c

```diff
@@ -595,45 +588,23 @@ static bool sup_drive_mode_init(void)
     }
     else
     {
-        if (cfg.band_edge_lower == BAND_EDGE_LOWER &&
-            cfg.band_edge_guard == BAND_EDGE_GUARD &&
-            cfg.band_edge_upper == BAND_EDGE_UPPER &&
-            cfg.chan_step_size == 100 &&
-            cfg.chan_step_offset < 50)
-        {
-            return true;
-        }
-        else
-        {
-            return false;
-        }
+        return cfg.chan_step_size == 100 && cfg.chan_step_offset < 50;
```

Comment:
The band limits should already be set at this point. Drive-by Mode should only be defined by the channel step and the offset.